### DISCUSSION TITLE
TRV602Z - when will be added

### DISCUSSION THREAD

**Discussion by sabatpoznan** [Koenkk/zigbee2mqtt#24533] [Devices]
Hi, when TRV602Z thermostat will be added to zigbee2mqtt ? 

https://github.com/Koenkk/zigbee-herdsman-converters/pull/8170/files#diff-0157bb612c63e92c50f93116e1f50bf8e263030ed26d86caa28479159c2a9f4aL5121

I can see it is here, but my zigbee2mqtt still shows unsupported

**Comment by Koenkk** [comment]
Next release is on 1 November!
